Hunting Queries/TeamsLogs/NewBotAdded.yaml

```diff
@@ -0,0 +1,27 @@
+id: bf76e508-9282-4cf1-9cc1-5c20c3dea2ee
+name: Previously unseen bot or applicaiton added to Teams
+description: |
+  'This hunting query helps identify new, and potentially unapproved applications or bots 
+  being added to Teams.
+  This query requires you to have Teams data collected from the O365 Management Activity API
+  and the Teams parser function enabled.'
+tactics:
+  - Persistance
+  - Collection
+relevantTechniques:
+  - T1176
+  - T1119
+query: |
+
+  // If you have more than 30 days worth of Teams data change this value
+  let data_date = 30d;
+  let historical_bots = (
+  TeamsData
+  | where TimeGenerated > ago(data_date)
+  | where isnotempty(AddOnName)
+  | project AddOnName);
+  TeamsData
+  | where TimeGenerated > ago(1d)
+  // Look for add-ins we have never seen before
+  | where AddOnName in (historical_bots)
+  | extend timestamp = OfficeActivity_TimeGenerated, AccountCustomEntity = UserId
```

Comment:
There is no OfficeActivity_TimeGenerated, change to timestamp = TimeGenerated.

Reply:
Fixed